api/src/task/ecs_background_task.py

```diff
@@ -48,11 +48,13 @@ def entrypoint(db_session: db.Session):
     def decorator(f: Callable[P, T]) -> Callable[P, T]:
         @wraps(f)
         def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
-            with _ecs_background_task_impl(task_name):
-                # Finally execute the function with New Relic instrumentation
-                return newrelic.agent.background_task(name=task_name, group="Python/ECSTask")(f)(
-                    *args, **kwargs
-                )
+            # Wrap with New Relic instrumentation
+            application = newrelic.agent.register_application(timeout=10.0)
+            with newrelic.agent.BackgroundTask(application, name=task_name, group="Python/ECSTask"):
```

Comment:
FYI - I flipped the background task to wrap the ecs_background_task_impl method as it has some logs/operations it does that I want included in New Relic